### PR TITLE
LatestNews Home

### PR DESCRIPTION
**Pull request by ZeyadSultan**

### CODE REVIEW

File: e-store/e-store/ScenesScreen/LatestNews/LatestNewsCells/LatestNewsTableViewCell.swift [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+//  e-store
+//
+//  Created by Zeyad on 03/05/2023.
+//
+
+import UIKit
+
+class LatestNewsTableViewCell: UITableViewCell {
+    
+    @IBOutlet weak var newsTitle: UILabel!
```
Comment (Mahmoud-Bassuni) [inline e-store/e-store/ScenesScreen/LatestNews/LatestNewsCells/LatestNewsTableViewCell.swift:12]: just recommendation make it private

File: e-store/e-store/ScenesScreen/LatestNews/LatestNewsView.swift [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+//
+
+import UIKit
+
+class LatestNewsView: UIView {
+    
+    @IBOutlet weak var latestNewsTableView: UITableView!
+    
+    var dataSource:[LatestNewsModel] = []
+    var viewModel = LatestNewsViewModel()
```
Comment (Mahmoud-Bassuni) [inline e-store/e-store/ScenesScreen/LatestNews/LatestNewsView.swift:15]: plz declare  only as you set it on init

File: e-store/e-store/ScenesScreen/LatestNews/LatestNewsCells/LatestNewsTableViewCell.swift (hunk 1 above)
Comment (ZeyadSultan) [reply e-store/e-store/ScenesScreen/LatestNews/LatestNewsCells/LatestNewsTableViewCell.swift:12]: done

File: e-store/e-store/ScenesScreen/LatestNews/LatestNewsModel.swift [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+//  e-store
+//
+//  Created by Zeyad on 03/05/2023.
+//
+
+import Foundation
+import UIKit
+
+struct LatestNewsModel{
+    var newsImage: UIImage
```
[the remark this answers is not in the document]
Comment (ZeyadSultan) [reply e-store/e-store/ScenesScreen/LatestNews/LatestNewsModel.swift:12]: done

File: e-store/e-store/ScenesScreen/LatestNews/LatestNewsView.swift (hunk 2 above)
Comment (ZeyadSultan) [reply e-store/e-store/ScenesScreen/LatestNews/LatestNewsView.swift:15]: done

**Review by Mahmoud-Bassuni — APPROVED** [review]

**Review by AhmedAtef277 — APPROVED** [review]

**Review by hassanSayed0 — APPROVED** [review]

**Pull request merged**